src/partials/GoogleMap.js

```diff
@@ -28,6 +28,7 @@ export default class GoogleMap {
 		apiKey,
 		mapContainer,
 		hasCluster,
+		resultRendering,
```

Comment:
Sollte hier dann das gleiche Naming haben wie in der initialen config `hasCustomResults`.